datasets/modeled/datacenters/dataset.json5

```diff
@@ -0,0 +1,118 @@
+{
+  dataset_id: "decarb_2023_datacenters",
+  dataset_type: "modeled",
+  sector_description: "Residential and commercial",
+  description: "Datacenter load for the contiguous US for DECARB 2023 High Scenario, calculated using the EPRI 2024 whitepaper Analyzing Artificial Intelligence and Data Center Energy Consumption",
+  origin_creator: "Ashreeta Prasanna",
+  origin_organization: "NREL",
+  origin_contributors: [
+    "Ashreeta Prasanna",
+    "Elaine Hale",
+    "Sarah Smith"
+  ],
+  origin_project: "DECARB Year 1, Task 6",
+  origin_date: "2024-07-08",
+  origin_version: "v0.1.0",
+  data_source: "EPRI, LBL",
+  source: "Datacenter load for DECARB 2023",
+  data_classification: "low",
+  tags: [
+    "datacenters"
+  ],
+  trivial_dimensions: [
+    "scenario",
+    "subsector",
+    "weather_year",
+  ],
+  data_schema: {
+    table_format: {
+      format_type: "unpivoted",
+      value_column: "value",
+    },
+    data_schema_type: "one_table",
+  },
+
+  dimensions: [
+    {
+      type: "geography",
+      name: "State",
+      display_name: "State",
+      file: "../../../project/dimensions/supplemental/states.csv",
+      module: "dsgrid.dimension.standard",
+      class: "State",
+      description: "US States L48",
+    },
+    {
+      type: "metric",
+      name: "Datacenters",
+      display_name: "Datacenters",
+      file: "dimensions/enduses.csv",
+      module: "dsgrid.dimension.standard",
+      class: "EnergyEndUse",
+      description: "Unspecified end use electricity",
+    },
```

Comment:
Here, and for the residuals, I believe you are just using the "Unspecified Electricity" end use, in which case I would call this dimension "Unspecified Electricity" and reuse it verbatim in all relevant locations.

Reply:
Furthermore, I'd be supportive of putting dimensions like this in a `datasets/modeled/dimensions` folder. (To clarify that those dimensions are reused across datasets, but are not part of the project dimensions.)